pkg/cmd/job/job_history.go

```diff
@@ -62,9 +63,11 @@ func newCmdJobHistory(ctx api.Context) *cobra.Command {
 			}
 
 			fmt.Fprintln(ctx.Out(), historyMessage(job, failures))
-			table := cli.NewTable(ctx.Out(), []string{"TASK ID", "STARTED", "FINISHED"})
+
+			tableTitle := []string{"JOB ID", "STARTED", "FINISHED", "TASK ID(S)"}
+			table := cli.NewTable(ctx.Out(), tableTitle)
 			for _, run := range runs {
-				table.Append([]string{run.ID, run.CreatedAt, run.FinishedAt})
+				table.Append([]string{run.ID, run.CreatedAt, run.FinishedAt, strings.Join(run.Tasks, ", ")})
```

Comment:
Is there a limit on the number of task IDs we'll print here? How many are there usually, eg on a soak cluster?

Reply:
Maybe it is enough to put the task ids in the JSON output only.

Reply:
Tested on the soak cluster: I have only seen 1 task in the slice.

Reply:
Is it like this for all jobs which ran a lot of times? I feel like there is an unbounded number of task IDs here and in some cases it would make the table unreadable, but that's just an impression.

@rgo3 do you know more about this?

Reply:
Metronome starts a new Mesos task every time it starts a job, I expect the value to be 1 all the time but we can ask the Orchestration team:
```
➜  dcos-core-cli (1.14-patch.x) ✗ build/darwin/dcos job history yolo
        JOB ID                   STARTED                       FINISHED                                     TASK ID(S)
  20190614100626bV03V  2019-06-14T10:06:26.808+0000  2019-06-14T10:07:57.874+0000  yolo_20190614100626bV03V.429027bf-8e8c-11e9-abc1-9e6c55c7b873
  20190614100526UV4uj  2019-06-14T10:05:26.788+0000  2019-06-14T10:05:46.795+0000  yolo_20190614100526UV4uj.f46c090e-8e8b-11e9-abc1-9e6c55c7b873
  20190614100326eV9yF  2019-06-14T10:03:26.748+0000  2019-06-14T10:05:36.001+0000  yolo_20190614100326eV9yF.eddca72d-8e8b-11e9-abc1-9e6c55c7b873
  20190614100426fEC1t  2019-06-14T10:04:26.768+0000  2019-06-14T10:05:35.996+0000  yolo_20190614100426fEC1t.eddc590c-8e8b-11e9-abc1-9e6c55c7b873
  201906141001265PjAc  2019-06-14T10:01:26.708+0000  2019-06-14T10:03:24.867+0000  yolo_201906141001265PjAc.9fb8887a-8e8b-11e9-abc1-9e6c55c7b873
  20190614100226TfSVW  2019-06-14T10:02:26.728+0000  2019-06-14T10:03:24.862+0000  yolo_20190614100226TfSVW.9fb8fdab-8e8b-11e9-abc1-9e6c55c7b873
  201906141000263sfQi  2019-06-14T10:00:26.688+0000  2019-06-14T10:01:13.830+0000  yolo_201906141000263sfQi.519490d8-8e8b-11e9-abc1-9e6c55c7b873
  20190614095926jxcH1  2019-06-14T09:59:26.668+0000  2019-06-14T10:01:13.817+0000  yolo_20190614095926jxcH1.5194def9-8e8b-11e9-abc1-9e6c55c7b873
  20190614095826pVNOI  2019-06-14T09:58:26.648+0000  2019-06-14T09:59:02.692+0000  yolo_20190614095826pVNOI.03702406-8e8b-11e9-abc1-9e6c55c7b873
  20190614095726pH8Gy  2019-06-14T09:57:26.628+0000  2019-06-14T09:59:02.687+0000  yolo_20190614095726pH8Gy.03707227-8e8b-11e9-abc1-9e6c55c7b873
  20190614095626DbX1J  2019-06-14T09:56:26.609+0000  2019-06-14T09:56:51.625+0000  yolo_20190614095626DbX1J.b54c2c64-8e8a-11e9-abc1-9e6c55c7b873
  20190614095526f1oSN  2019-06-14T09:55:26.588+0000  2019-06-14T09:56:51.518+0000  yolo_20190614095526f1oSN.b54c7a85-8e8a-11e9-abc1-9e6c55c7b873
  20190614095426kvsOt  2019-06-14T09:54:26.568+0000  2019-06-14T09:54:40.283+0000  yolo_20190614095426kvsOt.6728f813-8e8a-11e9-abc1-9e6c55c7b873
  20190614095226GBZEn  2019-06-14T09:52:26.528+0000  2019-06-14T09:54:29.493+0000  yolo_20190614095226GBZEn.60994812-8e8a-11e9-abc1-9e6c55c7b873
  20190614095326uJfSk  2019-06-14T09:53:26.548+0000  2019-06-14T09:54:29.487+0000  yolo_20190614095326uJfSk.6098f9f1-8e8a-11e9-abc1-9e6c55c7b873
  20190614095126sb71z  2019-06-14T09:51:26.508+0000  2019-06-14T09:52:18.364+0000  yolo_20190614095126sb71z.1275295f-8e8a-11e9-abc1-9e6c55c7b873
  20190614095026lrnLm  2019-06-14T09:50:26.488+0000  2019-06-14T09:52:18.357+0000  yolo_20190614095026lrnLm.12757780-8e8a-11e9-abc1-9e6c55c7b873
  20190614094926DtnuJ  2019-06-14T09:49:26.469+0000  2019-06-14T09:50:07.316+0000  yolo_20190614094926DtnuJ.c451a6ed-8e89-11e9-abc1-9e6c55c7b873
  20190614094826JczX0  2019-06-14T09:48:26.448+0000  2019-06-14T09:50:07.309+0000  yolo_20190614094826JczX0.c451f50e-8e89-11e9-abc1-9e6c55c7b873
  20190614094726B71Gv  2019-06-14T09:47:26.429+0000  2019-06-14T09:47:56.200+0000  yolo_20190614094726B71Gv.762f35ec-8e89-11e9-abc1-9e6c55c7b873
  20190614094626wR38n  2019-06-14T09:46:26.409+0000  2019-06-14T09:47:56.190+0000  yolo_20190614094626wR38n.762ee7cb-8e89-11e9-abc1-9e6c55c7b873
  20190614094526XDxBv  2019-06-14T09:45:26.388+0000  2019-06-14T09:45:44.915+0000  yolo_20190614094526XDxBv.280ac91a-8e89-11e9-abc1-9e6c55c7b873
  20190614094426icnMM  2019-06-14T09:44:26.373+0000  2019-06-14T09:45:34.317+0000  yolo_20190614094426icnMM.1fbe26d9-8e89-11e9-abc1-9e6c55c7b873
```

Reply:
I have asked the ochestration team:
>  We are playing it safe because other places in the API dealing with tasks also have arrays for this. There is a situation where task might fail but we're still in time period to retry it and we'll launch another task for the same run.

Reply:
Is there a need to display all tasks? Otherwise we might figure out a way to display the task that executed this job run successfully? E.g. the TASK ID that matches the RUN ID.

Reply:
There is no way to figure that out, the task IDs contain a random string. As we know the number of tasks for a given job run will nearly always be 1, I think it's fine to display all tasks all the time and not add logic.

Reply:
There is no way to figure that out, the task IDs contain a random string. As we know the number of tasks for a given job run will nearly always be 1, I think it's fine to display all tasks all the time and not add logic.